metrics/internal/cache/pv.go

```diff
@@ -135,6 +135,10 @@ func (p *PersistentVolumeStore) Add(obj interface{}) error {
 		}
 	}
 
+	if p.monitorConfig.clusterID != pv.Spec.CSI.VolumeAttributes["clusterID"] {
```

Comment:
@Madhu-1 is it safe to assume that `pv.Spec.CSI.VolumeAttributes["clusterID"]` always exists and points to the ceph cluster which backs the PV?

Reply:
@Madhu-1 Is this valid when using radosnamespace and subvolumegroups as well?